File: Client/cinema-app/src/app/core/models/movie/movie.ts
import { Genre } from '@core/models/genre/genre';
import { Image } from '@core/models/image/image';

export interface Movie {
  id: number
  title: string,
  description: string,
  genre: Genre,
  yearOfIssue: number,
  image: Image,
  showInCinemasStartDate: Date,
  showInCinemasEndDate: Date,
  durationInMinutes: number
}

File: Server/Cinema/CinemaApp.Domain/Entities/Movie.cs
namespace CinemaApp.Domain.Entities
{
    public class Movie
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime ShowInCinemasStartDate { get; set; }
        public DateTime ShowInCinemasEndDate { get; set; }
        public short DurationInMinutes { get; set; }
        public int GenreId { get; set; }
        public Genre Genre { get; set; }
        public long? ImageId { get; set; }
        public Image Image { get; set; }
        public ICollection<MovieSession> MovieSessions { get; set; }
    }
}